pkg/registry/registry.go

```diff
@@ -269,13 +279,11 @@ func (r *Registry) handleMirror(rw mux.ResponseWriter, req *http.Request, ref re
 			proxy := httputil.NewSingleHostReverseProxy(u)
 			proxy.Transport = r.transport
 			proxy.ErrorHandler = func(_ http.ResponseWriter, _ *http.Request, err error) {
-				log.Error(err, "proxy failed attempting next")
+				log.Error(err, "mirror request failed attempting next")
```

Comment:
Nitpick: this message is ambiguous, and should really be "mirror request failed, attempting next". Alternatively, you could include the new `mirrorAttempts` variable: "request to mirror %d failed".

Reply:
I agree with this, commas are important. I changed it and removed the attempting next because that would be a lie if max attempts was reached. Instead I changed the log to include the attempt count.